torch/csrc/jit/runtime/static/memory_planner.cpp

```diff
@@ -154,9 +155,13 @@ void MemoryPlanner::allocate() {
     void* src = static_cast<void*>(start + offset);
 
     for (auto* tensor : tensors) {
-      tensor->storage().set_data_ptr_noswap(
-          at::DataPtr(src, src, nullptr, tensor->device()));
-      tensor->storage().set_nbytes(tensor_size);
+      const at::Storage* storage = managed_tensor_storages_.empty() ? &tensor->storage() : managed_tensor_storages_[storageIdx++];
```

Comment:
should mark it as C10_UNLIKELY that managed_tensor_storages_ is empty since that only applies on the first pass